### ISSUE TITLE
[master] setPosition - Cannot match against 'undefined' or 'null'.

### ISSUE THREAD

**Issue by Sir-Will** [hydrabolt/discord.js#1918]
**Please describe the problem you are having in as much detail as possible:**
Just updated to the master branch because of a bug I needed a fix for.
I'm getting the error `Cannot match against 'undefined' or 'null'.` since then with the code below.
```
  setPosition(position, { relative, reason }) {
             ^
```

**Include a reproducible code sample here, if possible:**
```js
const bot	= new Discord.Client();
bot.channels.forEach(function (channel) {
	if (channel.type != 'voice') return;
	channel.setPosition(channel.rawPosition + 1)
})
```

**Further details:**

- discord.js version: master
- node.js version: 8.2.1
- Operating system: Ubuntu

- [ ] I found this issue while running code on a __user account__
- [x] I have also tested the issue on latest master, commit hash: c46c092d0de1d556c71d26bc3de45fb481dd75df

**Comment by robflop** [comment]
There is no ``rawPosition`` property for channels, meaning you're doing ``undefined + 1`` which results in NaN.

**Comment by Sir-Will** [comment]
There is. `console.log(channel)` or `console.log(channel.rawPosition)` shows it. It was renamed from `position` to `rawPosition` as far as I can see.

**Comment by robflop** [comment]
Yeah, I see now. The documentation wasn't updating. 

Could you provide the commit hash as requested in the issue template? You only ticked the box, yet did not fill out the hash.

**Comment by Sir-Will** [comment]
Ups, missed that. I added it.

**Comment by robflop** [comment]
Thanks, checking it out now.

**Comment by robflop** [comment]
It was indeed a problem with the discord.js source. It's fixed in #1909 so you'll have to wait for that to be merged.

**Comment by Sir-Will** [comment]
Okay, thanks.
